src/get-started/java-binding.md

```diff
@@ -31,16 +31,15 @@ deployment.
   ```
 
 - Install [Libsodium][libsodium] as the necessary runtime dependency.
-  For Mac OS also install RocksDB library:
-  
+
 ??? example "Linux (Ubuntu)"
     ```bash
     sudo apt-get update && sudo apt-get install libsodium-dev
```

Comment:
@dip56 @vitvakatu if the Rocksdb is linked dynamically on Linux and is not distributed with the app, why isn't it required to run a packaged application?

Reply:
Well, that's because our packaged app is linked statically on Linux 🤷‍♂ That's because on Linux it's not necessary to set ROCKSDB_STATIC=1, it's enough to set `ROCKSDB_LIB_DIR`

Reply:
If statically, then the instructions are good.